src/sentry/static/sentry/app/components/platformicon.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const PLATFORM_TO_ICON = {
+  apple: 'apple',
+  cocoa: 'apple',
+  cordova: 'cordova',
+  csharp: 'csharp',
+  elixir: 'elixir',
+  electron: 'electron',
+  go: 'go',
+  java: 'java',
+  'java-android': 'java',
+  'java-appengine': 'app-engine',
+  'java-log4j': 'java',
+  'java-log4j2': 'java',
+  'java-logback': 'java',
+  'java-logging': 'java',
+  javascript: 'javascript',
+  'javascript-angular': 'angularjs',
+  'javascript-backbone': 'javascript',
+  'javascript-ember': 'ember',
+  'javascript-react': 'react',
+  'javascript-vue': 'vue',
+  node: 'nodejs',
+  'node-connect': 'nodejs',
+  'node-express': 'nodejs',
+  'node-koa': 'nodejs',
+  'objective-c': 'apple',
+  perl: 'perl',
+  php: 'php',
+  'php-laravel': 'laravel',
+  'php-monolog': 'php',
+  'php-symfony2': 'php',
+  python: 'python',
+  'python-flask': 'flask',
+  'python-bottle': 'bottle',
+  'python-celery': 'python',
+  'python-django': 'django',
+  'python-pylons': 'python',
+  'python-pyramid': 'python',
+  'python-rq': 'python',
+  'python-tornado': 'python',
+  'react-native': 'apple',
+  ruby: 'ruby',
+  'ruby-rack': 'ruby',
+  'ruby-rails': 'rails',
+  rust: 'rust',
+  swift: 'swift',
+};
+
+export function getIcon(platform) {
+  let icon = PLATFORM_TO_ICON[platform];
+
+  if (!icon) {
+    return 'generic';
+  } else {
+    return icon;
+  }
+}
+
+const Platformicon = ({platform, size, width, height, ...props}) => {
```

Comment:
nit: can we rename this to `platformIcon` (and the file to be `platformIcon`) to be consistent?

Reply:
The lib is 'platformicons' (all one word).

Reply:
Oh, but I see that we have StyledPlatformIcon, etc. I'll fix.